.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: PowervRA
+
+on:
+  pull_request:
+    paths:
+      - src/**
+    branches:
+      - master
+
+env:
+  tf_version: 'latest'
+  tg_version: 'latest'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Are these AWS keys in here by mistake or are they being used for something?

Reply:
A stray commit. I've been looking at GitHub actions as an option for our builds and releases.

I'll remove the file shortly.